src/reasonJs.re

```diff
@@ -2,11 +2,13 @@ external requestAnimationFrame : (unit => unit) => unit = "" [@@bs.val];
 
 module Base64 = Base64Re;
 module Gl = GlRe;
+module Canvas2d = Canvas2d;
```

Comment:
Can you name the module `Canvas2dRe`? This is to keep internal modules from conflicting with modules defined in its dependents. You should still keep the alias as it is, though, so it won't affect how dependents actually consume it.